lib/team.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/player'
+
+class Team 
+  
+  attr_reader       :country,
+                    :players,
+                    :eliminated 
+  
+  def initialize(country)
+    @country = country
+    @players = []
+    @eliminated = false 
+  end 
+  
+  def eliminated?
+    @eliminated 
+  end 
+  
+  def eliminated
```

Comment:
Will likely have a collision with the `eliminated` attr_reader` that you created above. Believe this is overwriting that method. Switch your `attr_reader` to an `attr_writer` then you can do away with this method.